app/src/molecules/WizardHeader/index.tsx

```diff
@@ -31,24 +33,56 @@ const EXIT_BUTTON_STYLE = css`
     opacity: 70%;
   }
 `
+const ON_DEVICE_BUTTON_STYLE = css`
+  margin-right: 1.75rem;
+  font-size: 1.375rem;
+  font-weight: 700;
+  text-transform: ${TYPOGRAPHY.textTransformCapitalize};
+  color: ${COLORS.darkGreyEnabled};
+`
+
 export const WizardHeader = (props: WizardHeaderProps): JSX.Element => {
   const { totalSteps, currentStep, title, onExit, exitDisabled } = props
+  const isOnDevice = useSelector(getIsOnDevice)
   const { t } = useTranslation('shared')
 
   return (
     <Box backgroundColor={COLORS.white}>
       <Flex
-        padding={`${String(SPACING.spacing4)} ${String(SPACING.spacing6)}`}
+        padding={
+          isOnDevice
+            ? `28px`
+            : `${String(SPACING.spacing4)} ${String(SPACING.spacing6)}`
```

Comment:
Sorry this would be my fault, I added `String` before but Brian told me that we wouldn't need to add it to avoid the warning.

```suggestion
            : `${SPACING.spacing4} ${SPACING.spacing6}`
```